### PR TITLE
docs: README networks section

### PR DESCRIPTION
**Pull request by rootulp**
Closes https://github.com/celestiaorg/networks/issues/159

## Motivation
I'm not sure where the source of truth is for the version(s) of software that are running on our networks. This repo seems like a good candidate for such a table. I'm concerned this table won't be updated when software on our networks is updated so please comment if there's a better repo for such a table.

### CODE REVIEW

**Comment by rootulp** [comment]
Tagging @sysrex @Bidon15 for review

File: README.md [hunk 1]
```diff
@@ -4,3 +4,9 @@ This repository contains the `genesis.json` for Celestia testnets.
 
 Please refer to the Celestia Docs for guides on getting started running
 your own node [here](https://docs.celestia.org)
+
+## Networks
+
+Network | celestia-app | celestia-node
+--- | --- | ---
+Arabica | [v0.10.0-rc1](https://github.com/celestiaorg/celestia-app/releases/tag/v0.10.0-rc1) | [v0.5.0-rc5](https://github.com/celestiaorg/celestia-node/releases/tag/v0.5.0-rc5)
```
Comment (Bidon15) [inline README.md:12]: Suggestion: We def need to add sdk and core versions

Comment (rootulp) [reply README.md:12]: We can add them to this table but I'm curious why? Would they be more convenient than referencing the source of truth in celestia-app's `go.mod`. E.g:
1. https://github.com/celestiaorg/celestia-app/blob/124e53434685ada350162dc92133f5f83d2eb626/go.mod#L183
1. https://github.com/celestiaorg/celestia-app/blob/124e53434685ada350162dc92133f5f83d2eb626/go.mod#L185

Comment (rootulp) [reply README.md]: resolved in https://github.com/celestiaorg/networks/pull/162/commits/6fa44c71cab280a59db92669f66b5eb7c02a63cf

File: README.md [hunk 2]
```diff
@@ -4,3 +4,9 @@ This repository contains the `genesis.json` for Celestia testnets.
...
+
+## Networks
+
+| Network | celestia-node                                                                      | celestia-app                                                                        | celestia-core                                                                                      | cosmos-sdk                                                                                      |
+| ------- | ---------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------- |
+| Arabica | [v0.5.0-rc5](https://github.com/celestiaorg/celestia-node/releases/tag/v0.5.0-rc5) | [v0.10.0-rc1](https://github.com/celestiaorg/celestia-app/releases/tag/v0.10.0-rc1) | [v1.5.0-tm-v0.34.20](https://github.com/celestiaorg/celestia-core/releases/tag/v1.5.0-tm-v0.34.20) | [v1.4.0-sdk-v0.46.0](https://github.com/celestiaorg/cosmos-sdk/releases/tag/v1.4.0-sdk-v0.46.0) |
```
Comment (rootulp) [inline README.md:12]: [question for reviewers] where can I find what versions of our software are running on Mamaki?

**Comment by rootulp** [comment]
I don't have permission to add reviewers or assign myself on this PR. @MSevey may I be granted permissions? If not:

Reviewers: @Bidon15 @sysrex 
Assignee: @rootulp

**Comment by MSevey** [comment]
Is this still valid?

**Comment by rootulp** [comment]
Nope this PR is outdated and IDK how to update it b/c I'm not sure where the source of truth is for versions of our software across testnets. @sysrex @Bidon15 if ya'll can point me to where to look, I can fetch version numbers and maintain a markdown table like this.

**Pull request closed without merging**